wp-cache-phase1.php

```diff
@@ -187,7 +189,7 @@ function wp_cache_serve_cache_file() {
 			$wp_cache_home_path = '/';
 
 		// make sure ending slashes are ok
-		if ( $wp_cache_request_uri == $wp_cache_home_path || ( $wp_cache_slash_check && substr( $wp_cache_request_uri, -1 ) == '/' ) || ( $wp_cache_slash_check == 0 && substr( $wp_cache_request_uri, -1 ) != '/' ) ) {
+		if ( $wp_cache_request_uri == $wp_cache_home_path || ( $wp_cache_slash_check && substr( $wp_cache_request_uri, -1 ) == '/' ) || ( $wp_cache_slash_check == 0 && substr( $wp_cache_request_uri, -1 ) != '/' ) || do_cacheaction( 'serve_supercache_file', false ) ) {
```

Comment:
Ah, that's a good idea. I initially rewrote that line into multiple variables so that the condition itself would read like:

`if ( $is_home_page || $request_meets_slashed_expectation || $request_meets_unslashed_expectation || $serve_supercache_file ) {`

...but then I worried about making too many changes to the core plugin, so I reverted it back. Changing them to be on separate lines is a great middle-ground, though.